ecommers/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -10,6 +10,7 @@
 		1498D2341E8E89220040F4C2 /* GeneratedPluginRegistrant.m in Sources */ = {isa = PBXBuildFile; fileRef = 1498D2331E8E89220040F4C2 /* GeneratedPluginRegistrant.m */; };
 		3B3967161E833CAA004F5970 /* AppFrameworkInfo.plist in Resources */ = {isa = PBXBuildFile; fileRef = 3B3967151E833CAA004F5970 /* AppFrameworkInfo.plist */; };
 		535525C6E0F848FD27D5F8AB /* Pods_Runner.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 5DA8922DECF7128B1058B009 /* Pods_Runner.framework */; };
+		6E2F8E13244F13C1006BD932 /* GoogleService-Info.plist in Resources */ = {isa = PBXBuildFile; fileRef = 6E2F8E12244F13C1006BD932 /* GoogleService-Info.plist */; };
```

Comment:
do we need these changes in ios and android projects?

Reply:
Sure, this is one requirement to use Firebase